File: php/structure-in-rte/RegisterResolver.php
<?php
// DocSection: structure_in_rte_register_resolver
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
$client = new DeliveryClient("<YOUR_ENVIRONMENT_ID>");
$client->inlineLinkedItemsResolver= new CustomLinkedItemsResolver(); 
// EndDocSection
?>

File: java/structure-in-rte/RegisterResolver.java
// DocSection: structure_in_rte_register_resolver
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
DeliveryClient client = new DeliveryClient("<YOUR_ENVIRONMENT_ID>");
client.registerInlineContentItemsResolver(new TweetInlineContentItemsResolver());
// EndDocSection